imageroot/templates/config-site.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
Is this template needed? The container entry point script accepts environment variables!

Sadly I can't find a link to the script source code (btw, is it OSS?), anyway look at `/start.sh` to see if it can do the work by itself.

Reply:
https://bitbucket.org/jsuto/piler/src/master/docker/start.sh

and 

https://bitbucket.org/jsuto/piler/src/master/docker/


We miss a lot environment  variables for piler, maybe jsuto could accept PR from us